gui_qt/check_report.py

```
@@ -340,18 +348,15 @@ def idle_writeback_summary_for_work_mode(mode) -> WritebackSummary:
             message = (
-                "同步翻译可能直接写回项目文件，行为由 gemini_translate.py 决定；"
-                "请在副本或备份上验证，详情见诊断页。"
+                "同步翻译可能直接改项目文件；"
+                "请先在副本或备份上试跑，详情见诊断页。"
             )
         elif spec.mode == WorkMode.KEYWORD_EXTRACTION:
-            message = "关键词模式只生成候选报告，不会启用普通「写回翻译」按钮。"
+            message = "关键词模式只生成报告，不会修改游戏脚本。"
         elif spec.mode == WorkMode.REVISION:
-            message = "订正写回将使用 preview-revisions / apply-revisions 独立流程，不经过普通 check/apply。"
+            message = "订正写回与普通翻译分开；请先在「订正写回」页预览并确认。"
```

Ruff (0.15.18)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)